services/osslifecycle/osslifecycle.service.js

```diff
@@ -87,17 +73,15 @@ export default class OssTracker extends BaseService {
     }
   }
 
-  async fetch({ user, repo, branch }) {
+  async fetch({ fileUrl }) {
     return this._request({
-      url: `https://raw.githubusercontent.com/${user}/${repo}/${branch}/OSSMETADATA`,
+      url: fileUrl,
     })
   }
 
-  async handle({ user, repo, branch }) {
+  async handle({ user, repo, branch }, { file_url: fileUrl = '' }) {
```

Comment:
```suggestion
  async handle(pathParams, { file_url: fileUrl = '' }) {
```
We have to put something here because `queryParams` is the second argument to `handle()` but we're not actually using `user`, `repo` or `branch` in this function.